README.md

```diff
@@ -556,6 +559,48 @@ module.exports = buildWebpackConfig([
   workerLoader(),
 ])
 ```
+
+```js
+// src/workers/myawesome.worker.ts
+
+import { get } from 'lodash'
+// you can import modules in this worker
+
+const ctx: Worker = self as any
+
+// Listen to message from the parent thread
+ctx.addEventListener('message', event => {
+  console.log(event)
+  // Post data to parent thread
+  ctx.postMessage({ data: 'maronn' })
+})
+```
+
+```js
+// src/typings/custom/index.d.ts
```

Comment:
This is the main `.d.ts` path, please use this:

https://github.com/accurat/accurapp/blob/cf207a0589cff6f5b13e7bd4dcffa4fb05a89e26/packages/accurapp-scripts/utils/verifyTypeScriptSetup.js#L48